integration_test/tests/wallet.rs

```diff
@@ -484,6 +484,17 @@ fn wallet__keypool_refill() {
     let _: () = node.client.key_pool_refill().expect("keypoolrefill");
 }
 
+#[test]
+fn wallet__list_labels__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    let label = "list-label-test";
+    let _ = node.client.new_address_with_label(label).expect("newaddress");
+
+    let json: ListLabels = node.client.list_labels().expect("listlabels");
+
+    assert!(json.0.contains(&label.to_string()));
```

Comment:
I'm going to ack/merge this but in case it needs a rebase this line is a bit off. The `label` var is of type `&str`. This line allocates memory, creates an `String`, then immediately dereferences it to get an `&str`. So we can just pass in `label`.

Reply:
Ha I merged in order of PR number, got all the way to here!

Reply:
> The `label` var is of type `&str`. This line allocates memory, creates an `String`, then immediately dereferences it to get an `&str`. So we can just pass in `label`.

So... I had to change it to `any()` with a closure. With `assert!(json.0.contains(label));` I get the error ```expected `&String`, found `&str` ``` which is why I changed it to the above originally. I didn't know that you could compare a `String` to `&str` with the `==` in the closure but not with `contains()`.

Reply:
Bizzare, I thought I tested my suggestion yesterday but I guess I did something wrong (probably ran the `test17` command from crate root, thereby not _actually_ building the test).

And thanks, TIL about using `any` - its literally in the docs of `contains`!